backend/functions/me/me.function.ts

```diff
@@ -1,20 +1,37 @@
-import { AzureFunction, Context, HttpRequest } from '@azure/functions';
-import ContextCreator from '../lib/adapters/utils/application-context-creator';
-import { httpError, httpSuccess } from '../lib/adapters/utils/http-response';
+import { app, HttpRequest, HttpResponseInit, InvocationContext } from '@azure/functions';
+import ContextCreator from '../azure/application-context-creator';
+import { toAzureError, toAzureSuccess } from '../azure/functions';
+import { httpSuccess } from '../lib/adapters/utils/http-response';
 
-const httpTrigger: AzureFunction = async function (
-  functionContext: Context,
+const MODULE_NAME = 'ME-FUNCTION';
+
+export async function handler(
   request: HttpRequest,
-): Promise<void> {
-  const context = await ContextCreator.applicationContextCreator(functionContext, request);
+  functionContext: InvocationContext,
+): Promise<HttpResponseInit> {
+  const applicationContext = await ContextCreator.applicationContextCreator(
+    functionContext,
+    request,
+  );
   try {
-    context.session = await ContextCreator.getApplicationContextSession(context);
-    const response = { success: true, body: context.session };
-    functionContext.res = httpSuccess(response);
-  } catch (camsError) {
-    context.logger.camsError(camsError);
-    functionContext.res = httpError(camsError);
+    applicationContext.session =
```

Comment:
Just a note that the context in this case does not actually need the logger added. We could still add it though, because this may be the only function that would never need it, and the logger argument being optional could present bugs in the future.